fsspec/spec.py

```diff
@@ -68,7 +69,10 @@ def __call__(cls, *args, **kwargs):
     # TODO: it should be possible to disable this
     import pyarrow as pa
 
-    up = pa.filesystem.DaskFileSystem
+    if LooseVersion(pa.__version__) < LooseVersion("2.0"):
```

Comment:
Would it be better to use an `else` clause for this, rather than putting it in the `try` block, so that the code in the `try` block is minimized?